### ISSUE TITLE
Top Slabs Block AI Pathing

### ISSUE THREAD

**Issue by PfluorescentZebra** [TangoTek/TekTopia-Community#80]
**Describe the bug**
Merchants cannot find a path through an area of top slabs; nomads were able to path through the same space without issues.  

**To Reproduce**
1. Define a clear path into the village
2. Replace entrance blocks with top slabs only.
3. In the morning, receive error message in client chat that Merchant cannot create path to village.
4. Replace slabs with full blocks.
5.  Merchant will path correctly on the next morning.

**Expected behavior**
Merchant should be able to path through same spaces as Nomads.

** Server/Location **
What server were you playing on?  What If Server 1
What are the world coordinates of your town hall marker? 2681 / 116 / -8672

**Comment by JohnyDL** [comment]
Possibly related to #51

**Comment by PfluorescentZebra** [comment]
Johny, that issue seems to be related to stairs and blocking the air spaces above partial blocks.  The half slabs were on a plain, on the surface, and had no blocks above them.  The issue was not in the space, but in the inability for only the merchant to use top slabs.  He uses full blocks at the same y level without changing anything else.

**Comment by JohnyDL** [comment]
I'm just trying to keep things that might have similar bug origins together if I'm wrong that's okay but connecting them should, in theory, help Tango track down things that might be interrelated and deal with them faster together than dealing with one bug at a time

**Comment by PfluorescentZebra** [comment]
Yes, I see that. The merchants are able to path over stairs with a 5 block height above them as well as bottom slabs. In all other aspects of the game, a top slab behaves as a full block. Without being able to see the source code, it is well neigh impossible to guarantee that the two issues are related. I felt it safer to create a separate ticket so nothing would be missed.

**Comment by JohnyDL** [comment]
I know which is why I said related to not duplicate of ;)
